cmd/zt_set_properties_test.go

```diff
@@ -0,0 +1,540 @@
+// Copyright © 2017 Microsoft <wastore@microsoft.com>
```

Comment:
Try adding test cases for cases when user passes wrong value of flag. Will the code be able to handle that?
For example: What if the user passes wrong fromTo?